aspnetcore/signalr/publish-to-azure-web-app.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Publish an ASP.NET Core SignalR app to Azure Web App
+author: rachelappel 
+description: Publish an ASP.NET Core SignalR app to Azure Web App
+manager: wpickett
```

Comment:
Add the following metadata under the `manager` metadata:

`monikerRange: '>= aspnetcore-2.1'`

This causes the doc to display only when 2.1 (or later) is selected from the version selector.